src/Wizard/Wizard.component.js

```diff
@@ -98,7 +123,13 @@ const Wizard = React.createClass({
             .filter(({step}) => step.visible !== false);
     },
 
-    _renderButtons(step, showPrevious, showNext) {
+    _renderButtons(step, showPrevious, showNext, showHelp) {
+        const help = (
```

Comment:
No need to change it here (the view is so trivial), but conditional views (which may be rendered or not) may go in a function so they are created only if necessary. Something like: `const help = () => ...`.